src/modes/simple_select.js

```diff
@@ -69,6 +69,10 @@ module.exports = function(ctx, startingSelectedIds) {
     });
   };
 
+  var emitModify = function() {
+    ctx.map.fire('draw.modified', { features });
```

Comment:
`features` should be dropped from this file and we should use what is currently selected.